### PR TITLE
Add API v1 support

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Change the API endpoint `/compute` to `/v1`

### CODE REVIEW

File: cmd/config_add.go [hunk 1]
```diff
@@ -184,7 +184,10 @@ Let's start over.
 		}
 	}
 
-	account.DNSEndpoint = strings.Replace(account.Endpoint, "/compute", "/dns", 1)
+	account.DNSEndpoint = strings.Replace(account.Endpoint, "/"+defaultAPIVersion, "/dns", 1)
+	if strings.Contains(account.DNSEndpoint, "/"+defaultLagacyAPIVersion) {
+		account.DNSEndpoint = strings.Replace(account.DNSEndpoint, "/"+defaultLagacyAPIVersion, "/dns", 1)
+	}
```
Comment (pierre-emmanuelJ) [inline cmd/config_add.go]: I think I can make a function for that

File: cmd/config.go [hunk 2]
```diff
@@ -75,8 +75,10 @@ func (a account) IsDefault() bool {
 }
 
 const (
+	defaultLagacyAPIVersion  = "compute"
```
Comment (falzm) [inline cmd/config.go]:
```suggestion
	legacyAPIVersion  = "compute"
```

File: cmd/config.go [hunk 3]
```diff
@@ -75,8 +75,10 @@ func (a account) IsDefault() bool {
...
+	defaultAPIVersion        = "v1"
```
Comment (falzm) [inline cmd/config.go]:
```suggestion
	apiVersion        = "v1"
```

**Comment by falzm** [comment]
[ch7114]

**Pull request merged**